src/tasks/__tests__/processBulkUploadTask.int.test.ts

```diff
@@ -111,26 +94,10 @@ const createTestBaseFields = async (): Promise<void> => {
 	});
 };
 
-const mockS3GetObjectReplyWithFile = async (file: File, filePath: string) =>
-	nock(await getS3Endpoint(file.s3Bucket.endpoint, file.s3Bucket.name))
-		.get(getS3KeyPath(file.s3Bucket.name, file.storageKey))
-		.query({ 'x-id': 'GetObject' })
-		.replyWithFile(200, filePath);
-
-const mockS3ResponsesForBulkUploadTaskProcessing = async (
-	bulkUploadTask: BulkUploadTask,
-	bulkUploadFilePath: string,
-) => {
-	const getRequest = await mockS3GetObjectReplyWithFile(
-		bulkUploadTask.proposalsDataFile,
-		bulkUploadFilePath,
-	);
-	return {
-		getRequest,
-	};
-};
-
 describe('processBulkUploadTask', () => {
+	beforeEach(() => {
+		s3Mock.reset();
```

Comment:
To avoid test interaction in a world where all tests can run concurrently, maybe one `s3Mock` per test instead of one per file. I have no idea how expensive that is, though, and for now it's OK.